src/plugins/intel_cpu/src/utils/blob_dump.cpp

```diff
@@ -176,7 +176,9 @@ void BlobDumper::dumpAsTxt(std::ostream &stream) const {
             for (size_t i = 0; i < data_size; i++) {
                 int i16n = blob_ptr[desc.getElementOffset(i)];
                 i16n = i16n << 16;
-                float fn = *(reinterpret_cast<const float *>(&i16n));
+                float fn;
```

Comment:
Can't we just work with bloat16_t type instead?

Reply:
Sorry @EgorDuplensky , I am not sure, if dump bfloat16_t type, it will not need to convert.
Hi @maxnick , I see you modified it. What's your idea? If we dump it with bf16 directly, or is it still readable ?

Reply:
`bfloat16_t ` has `operator float()` so I expect the number implicitly converted to float type and dumped normally.